src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -115,14 +121,41 @@ const NewQuestionnaireScreen = () => {
       questions: [],
       passScore: 0
     });
-    setLoading(false);
+    setTooltipMessage("");
+    setIsDisabled(true);
+  };
+
+  /**
+   * Function to check and set the tooltip message and isDisabled state
+   */
+  const updateTooltipAndIsDisabled = () => {
+    const isTitleEmpty = !questionnaire.title.trim();
+    const isDescriptionEmpty = !questionnaire.description.trim();
+    const isDisabled = isTitleEmpty || isDescriptionEmpty;
+
+    const tooltipMessage = isDisabled
+      ? isTitleEmpty && isDescriptionEmpty
+        ? strings.newQuestionnaireScreen.tooltipBothEmpty
+        : isTitleEmpty
+          ? strings.newQuestionnaireScreen.tooltipEmptyTitle
+          : strings.newQuestionnaireScreen.tooltipEmptyDescription
+      : "";
```

Comment:
Avoid nested ternaries. Maybe a switch or some if/if else conditions will be suitable here